Sources/Extensions/AppKit/NSViewExtensions.swift

```diff
@@ -141,4 +141,23 @@ extension NSView {
 	}
 
 }
+
+extension NSView
+{
+    /// SwifterSwift: set and get background color of NSView
+    @IBInspectable public var backgroundColor: NSColor? {
```

Comment:
@phiren Can you move this to be up above with the other properties?

Reply:
@SD10 , sure, but can you help me for adding unit test cases and change log?